src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -50,25 +52,34 @@ using Transport::Session;
     {                                                                                                                              \
         if (!::chip::ChipError::IsSuccess(expr))                                                                                   \
         {                                                                                                                          \
-            commandObj->AddStatus(commandPath, Protocols::InteractionModel::Status::code, #expr);                                  \
-            return true;                                                                                                           \
+            ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Protocols::InteractionModel::Status::code, #expr);                     \
+            return;                                                                                                                \
         }                                                                                                                          \
     } while (false)
 
 namespace {
 
-class GeneralCommissioningAttrAccess : public AttributeAccessInterface
+class GeneralCommissioningAttrAccess : public AttributeAccessInterface, public CommandHandlerInterface
```

Comment:
This is now more than AttrAccess. Suggest calling it GeneralCommissioningInstance